.github/workflows/ci_check.yml

```diff
@@ -67,6 +67,20 @@ jobs:
       - name: Rust Code Format
         run: ./scripts/workspace.py cargo fmt -- --check
 
+  toml-format:
```

Comment:
Hi, our PR should not include changes from other PRs. Please try using `merge` or `rebase` for this